### PR TITLE
Feed page download

### PR DESCRIPTION
**Pull request by reshetniktati**

### CODE REVIEW

File: client/src/views/Feed/Feed.vue [hunk 1]
```diff
@@ -72,8 +76,31 @@ export default {
...
+  },
   computed: {
     ...mapGetters('posts', ['getPosts'])
+  },
+  created () {
+    let route = ''
+    if (Authorized.isAuthorized()) {
+      route = '/api/feed/authorized'
+    } else {
+      route = '/api/feed/anonimus'
```
Comment (igorkokotko) [inline client/src/views/Feed/Feed.vue]: For such easy conditions you could use ternary operator to assign result to variable. - https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

File: server/controllers/feed.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+let getFeedAnonimus = function (req, res) {
+    try {
+        const collection = db.collection('posts').get()  
+        collection.then((querySnapshot) => {
+            let posts = {}
+            querySnapshot.docs.forEach((doc) => {
+                posts[doc.id] = doc.data();
+            })
+            res.json({posts})
```
Comment (igorkokotko) [inline server/controllers/feed.js]: Do you really need posts object? Or you can simply return res.json(posts)?

File: server/controllers/feed.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        const userDoc = db.collection('users').doc(req.userId)        
+        userDoc
+            .get()
+            .then(doc => {
+                let pref = doc.data().preferences
+
+                const postsDocs = db.collection('posts')
+                const postsCollection = postsDocs.where('tag', 'array-contains-any', pref).get()
+                postsCollection.then((querySnapshot) => {
+                    let posts = {}
```
Comment (igorkokotko) [inline server/controllers/feed.js:35]: What will happen if querySnapshot will be empty?

**Review by igorkokotko — CHANGES_REQUESTED** [review]

**Comment by igorkokotko** [comment]
@reshetniktati conflicts

File: client/src/views/Feed/Feed.vue (hunk 1 above)
Comment (reshetniktati) [reply client/src/views/Feed/Feed.vue]: Yes I could, but is it necessary? I think that this way is more readable

Comment (igorkokotko) [reply client/src/views/Feed/Feed.vue]: Ternary more readable for simple conditional statements.

**Review by igorkokotko — APPROVED** [review]

**Pull request merged**